fastchat/conversation.py

```diff
@@ -327,6 +341,21 @@ def register_conv_template(template: Conversation, override: bool = False):
 
 def get_conv_template(name: str) -> Conversation:
     """Get a conversation template."""
+    if (
+        "/" in name
+    ):  # We were given a model_path, so try to load in the respective tokenizer from Huggingface
+        try:
+            tokenizer = AutoTokenizer.from_pretrained(name)
```

Comment:
Try to avoid re-initialize it every time we call this.
Please note that this function will be called everytime for a new conversation